development/modules/auth/controllersAuth.js

```diff
@@ -46,7 +46,24 @@
     .controller("IndexPublicController", ["$scope", "Testimony", "NewsService", "CourseService",
       function ($scope, Testimony, NewsService, CourseService) {
 
-        $scope.showVideo = function (video) {
+          $scope.defaultImage = {
```

Comment:
Why is this assignation inside this method? That should be in the controller root.